### ISSUE TITLE
Putting themes in folders

### ISSUE THREAD

**Issue by RubikDash1013** [astronautlevel2/Anemone3DS#273]
I think it would be pretty cool to be able to put downloaded themes into folders, so, for example, you can separate NSFW and SFW themes. Or you could make folders based on characters or game series.

**Comment by Helloman892** [comment]
I agree, it would be pretty cool. Unfortunately, I can foresee it being a pretty low priority for the foreseeable future, as it would require a major overhaul of how the scrolling theme list works.

**Comment by RubikDash1013** [comment]
Oh ok, hopefully one day it can happen 😁

**Comment by Cdrow59** [comment]
I would love to see this feature! When managing a large number of themes it gets a little crazy sometimes.

**Comment by Cdrow59** [comment]
Would love to know a workaround as well. (Don't mind having to install separate homebrew just need a better solution.)
